Lib/distutils/command/build_ext.py

```diff
@@ -722,11 +722,4 @@ def get_libraries(self, ext):
             # Don't use the default code below
             return ext.libraries
         else:
-            from distutils import sysconfig
-            if sysconfig.get_config_var('Py_ENABLE_SHARED'):
-                pythonlib = 'python{}.{}{}'.format(
-                    sys.hexversion >> 24, (sys.hexversion >> 16) & 0xff,
-                    sysconfig.get_config_var('ABIFLAGS'))
-                return ext.libraries + [pythonlib]
-            else:
-                return ext.libraries
+            return ext.libraries
```

Comment:
I think you can remove `elif` branches above (for `aix` and `darwin`). It is better to leave comment instead of executing `elif` expressions every time. 

So we have difference with windows only, other systems are normal :-)

Reply:
`else` also can be removed. You can return in win32 branch if we have bad compiler and always return `ext.libraries` otherwise after `if` statement.

Reply:
Thanks!